packages/frontend/src/components/DashboardTiles/DashboardChartTile.tsx

```diff
@@ -724,12 +747,21 @@ const DashboardChartTileMain: FC<DashboardChartTileMainProps> = (props) => {
                                 label="Finish editing dashboard to use these actions"
                             >
                                 <Box>
-                                    {userCanManageChart && (
+                                    <Tooltip
+                                        disabled={
+                                            userCanManageChart || isEditMode
+                                        }
+                                        label={editButtonTooltipLabel}
+                                        position="top-start"
+                                    >
```

Comment:
if you wrap the `EditChartMenuItem` with a `Box` you can avoid the changes applied to `EditChartMenuItem` and `LinkMenuItem`, like so:

```tsx

<Tooltip
                                        variant="xs"
                                        disabled={
                                            userCanManageChart || isEditMode
                                        }
                                        label={editButtonTooltipLabel}
                                        position="top-start"
                                    >
                                        <Box>
                                            <EditChartMenuItem
                                                tile={props.tile}
                                                disabled={
                                                    isEditMode ||
                                                    !userCanManageChart
                                                }
                                            />
                                        </Box>
                                    </Tooltip>

```

See the `Box` component + see the `variant='xs'`

Reply:
Ohhhhh that's perfect!! Thank you 🎉